queries-src/injections.scm

```diff
@@ -1,6 +1,18 @@
 ((comment) @injection.content
   (#set! injection.language "comment"))
 
+(recipe_body
+  (shebang (language) @injection.language)
+  (#set! injection.include-children)
+) @injection.content
+
+(comment) @comment
+
+
+(source_file
+    ; (item (setting lang:(identifier) @language))
+    (recipe (recipe_body) @content)
+)
```

Comment:
FYI the scheme style has the closing parenthesis on the same line as whatever it closes. I know it looks pretty weird, but at least it's consistent.

Also a loose comment. No big deal, I'll fix this up before applying.

Reply:
This was an unintended addition when resolving merge conflicts, I'm going to remove since the point of this PR was to defer injections for things _without_ an explicit _and_ inside a recipe 😅 .